services/port_manager/src/main/java/com/futurewei/alcor/portmanager/processor/FixedIpsProcessor.java

```diff
@@ -263,6 +264,7 @@ private void fetchSubnetForAddCallBack(IRestRequest request) throws Exception {
             for (PortEntity portEntity : context.getPortEntities()) {
                 for (PortEntity.FixedIp fixedIp : portEntity.getFixedIps()) {
                     for (SubnetEntity subnetEntity : subnetEntities) {
+                        context.getPortRepository().createSubnetCache(fixedIp.getSubnetId());
```

Comment:
@DavidLiu506 Please give a comment here to describe why we put `createSubnetCache` here.